npm/design-system/src/components/searchInput/SearchInput.cy.tsx

```diff
@@ -2,9 +2,10 @@ import * as React from 'react'
 import { mount } from '@cypress/react'
 
 import { SearchInput } from './SearchInput'
-import { useCallback, useState } from 'react'
 import { mountAndSnapshot } from 'util/testing'
 
+const { useCallback, useState } = React
```

Comment:
Functionally the same -- no change in execution or tree shaking because `import * as React from 'react'` is done above.